great_expectations/rule_based_profiler/rule/rule.py

```diff
@@ -67,7 +63,7 @@ def generate(
                 parameter_builder.build_parameters(
                     parameter_container=parameter_container,
                     domain=domain,
-                    variables=self.variables,
+                    variables=variables,
```

Comment:
Thank you for this 🙇🏽 I much prefer this implementation with passing down state from RBP.

Reply:
@cdkini No -- thank you!  That was a major catch by you, and now it is so much better!  Thanks!